shared/panels/deposit-transaction-sidepanel/deposit-transaction-sidepanel.tsx

```diff
@@ -0,0 +1,109 @@
+import { useTranslation } from "react-i18next";
+
+import { CurrencyReactQueryAdapter } from "@/core/application/react-query-adapter/currency";
+
+import { Button } from "@/design-system/atoms/button/variants/button-default";
+import { Input } from "@/design-system/atoms/input";
+import { Typo } from "@/design-system/atoms/typo";
+import { Accordion, AccordionLoading } from "@/design-system/molecules/accordion";
+import { CardTemplate, CardTemplateLoading } from "@/design-system/molecules/cards/card-template";
+
+import { ErrorState } from "@/shared/components/error-state/error-state";
+import { SidePanelBody } from "@/shared/features/side-panels/side-panel-body/side-panel-body";
+import { SidePanelFooter } from "@/shared/features/side-panels/side-panel-footer/side-panel-footer";
+import { SidePanelHeader } from "@/shared/features/side-panels/side-panel-header/side-panel-header";
+import { useSidePanel, useSinglePanelData } from "@/shared/features/side-panels/side-panel/side-panel";
+import { useDepositTransactionSidepanel } from "@/shared/panels/deposit-transaction-sidepanel/deposit-transaction-sidepanel.hooks";
+
+export function DepositTransactionSidepanel() {
+  const { t } = useTranslation();
+  const { name } = useDepositTransactionSidepanel();
+  const { Panel } = useSidePanel({ name });
+  const { currencyId, networkName } = useSinglePanelData<{ currencyId: string; networkName: string }>(name) ?? {
```

Comment:
`{ currencyId: string; networkName: string }`

You can export this in to the deposition-transaction-sidepanel.types.ts file thanks we that you don't have to write it twice ( exemple on the project side panel )